src/internal/connector/graph_connector_disconnected_test.go

```diff
@@ -21,12 +21,16 @@ import (
 // Disconnected Test Section
 // ---------------------------------------------------------------
 type DisconnectedGraphConnectorSuite struct {
-	suite.Suite
+	tester.Suite
 }
 
 func TestDisconnectedGraphSuite(t *testing.T) {
+	s := &DisconnectedGraphConnectorSuite{
+		Suite: tester.NewUnitSuite(t, nil),
+	}
+
 	tester.LogTimeOfTest(t)
```

Comment:
Oh hey, we should call this in NewUnitSuite and NewIntegrationSuite.

Reply:
hmmm actually I'm not certain this will be useful if we call it when we make a new suite instance. It will only tell us when the entire suite started, not when each test started

Reply:
I'm not sure it's even useful as it stands 😅